src/Symfony/Component/Form/DependencyInjection/FormPass.php

```diff
@@ -62,6 +67,11 @@ private function processFormTypes(ContainerBuilder $container): Reference
             $commandDefinition->setArgument(2, array_keys($servicesMap));
         }
 
+        if ($csrfTokenIds && $container->hasDefinition('form.type_extension.csrf')) {
+            $csrfExtension = $container->getDefinition('form.type_extension.csrf');
+            $csrfExtension->replaceArgument(7, $csrfTokenIds);
```

Comment:
Oh wait, won't this break if the Form component is used with an older release of the FrameworkBundle where `form.type_extension.csrf` service doesn't have eight arguments?

```suggestion
            $csrfExtension->setArgument('$defaultTokenId', $csrfTokenIds);
```

Like this instead?

Reply:
ah right, should be good now!